README.md

```diff
@@ -1,22 +1,140 @@
-[![Build Status](https://travis-ci.org/gcovr/gcovr.png?branch=master)](https://travis-ci.org/gcovr/gcovr) 
+gcovr
+=====
 
-Gcovr Overview
---------
+generate GCC code coverage reports
+
+[website] • [documentation] • [bugtracker] • [GitHub][repo] • [![Build Status](https://travis-ci.org/gcovr/gcovr.png?branch=master)][travis-ci]
```

Comment:
Would you consider switching to the SVG badge format instead of PNG ? It's much more readable I think.
SVG: ![SVG](https://api.travis-ci.org/gcovr/gcovr.svg?branch=master) https://api.travis-ci.org/gcovr/gcovr.svg?branch=master
vs
PNG: ![PNG](https://api.travis-ci.org/gcovr/gcovr.png?branch=master) https://api.travis-ci.org/gcovr/gcovr.png?branch=master

Reply:
Fixed! TBH I can't see a difference unless I zoom in, but that's exactly why it's great to have PRs reviewed by another set of eyes :D